webroot/index-standalone-chat.html

```diff
@@ -0,0 +1,29 @@
+
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/>
+    <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet" />
+    <link href="./styles/chat.css" rel="stylesheet" />
+    <link href="./styles/standalone-chat.css" rel="stylesheet" />
+
+    <script src="//unpkg.com/showdown/dist/showdown.min.js"></script>
+
+  </head>
+
+  <body>
+
+    <div id="messages-only"></div>
+
+    <script type="module">
+      import { render, h } from 'https://unpkg.com/preact?module';
+      import htm from 'https://unpkg.com/htm?module';
+      const html = htm.bind(h);
+      import StandaloneChat from './js/app-standalone-chat.js';
+
+      (function () {
```

Comment:
nit: I tend not to use IIFEs in modules, since they are scoped and namespaced by default, you don't have to worry about namespace collisions or variables polluting the global namespace.